src/gui/properties/proptabbar.cpp

```diff
@@ -44,29 +44,34 @@ PropTabBar::PropTabBar(QWidget *parent) :
   m_btnGroup = new QButtonGroup(this);
   // General tab
   QPushButton *main_infos_button = new QPushButton(GuiIconProvider::instance()->getIcon("document-properties"), tr("General"), parent);
-  main_infos_button->setShortcut(QKeySequence(QString::fromUtf8("Alt+P")));
+  main_infos_button->setShortcut(QKeySequence(QString::fromUtf8("Ctrl+G")));
```

Comment:
I might be very well wrong, but to my understanding `Ctrl+` shortcuts are used for actions, while `Alt+` are for changing view or behaviour.  The difference might be negligible, but in this case the action (`main_infos_button`) clearly changes view.